### PR TITLE
DBZ-2105 Update to AsciiDoc markup that works upstream and downstream

### PR DESCRIPTION
**Pull request by TovaCohen**
These are the updates needed to reuse upstream content in downstream books. 
For `xref:` formatted cross-references, if they were in upstream-only content, I mostly left them in place.  More than 30 files in the `partials/modules` directory contain `xref:` cross-reference formats. I don't know how to address these yet. They should not be there, but for now, they will work downstream. 

In `architecture.adoc`, there is an `xref:` cross-reference to deprecated content. So I conditionalized the "Embedded Engine" section to be only upstream since I plan to reuse the `architecture.adoc` file in the downstream Debezium User Guide for the 2020-07 release.

### CODE REVIEW

File: documentation/modules/ROOT/pages/configuration/mongodb-event-flattening.adoc [hunk 1]
```diff
@@ -1,5 +1,6 @@
+[id="mongodb-new-document-state-extraction"]
 = MongoDB New Document State Extraction
```
Comment (Naros) [inline documentation/modules/ROOT/pages/configuration/mongodb-event-flattening.adoc:2]: On around Line 185, in the `MongoDB $unset handling` section, there is a reference to Debezium.

**Comment by TovaCohen** [comment]
@Naros  Your DBZ-2119 commit does not of course belong here. Can you remove it? Or tell me how? I think I pulled it into my local master and then had to pull it into my topic branch before I could push my topic branch to my fork. Sorry. 

I made the updates you asked for. 
I also revised the paragraph that had the instance of "Debezium" that needed to be replaced with {prodname}. Hope I got it right. 
And I had to add a link attribute to antora.yml so I took the opportunity to alphabetize the link attributes so I could find them more easily. 
Let me know if anything else needs to change.  Thanks!

**Comment by TovaCohen** [comment]
I added another commit that changes the column widths in tables of options and properties so that the downstream rendering is user-friendly.

**Comment by Naros** [comment]
> Can you remove it? Or tell me how? I think I pulled it into my local master and then had to pull it into my topic branch before I could push my topic branch to my fork. Sorry.

I could be mistaken but this looks like a merge commit rather than a rebase against master?  What you could try doing is an interactive rebase and specify `drop` for my commit from master.  Ideally, all that should leave is all the DBZ-2105 commits.  You may end up having to do a force push against your topic branch now however.

**Comment by TovaCohen** [comment]
Wow. I think that worked! I don't see the extra commit any more. Thanks!

**Comment by Naros** [comment]
Applied to master and 1.1.  Thanks @TovaCohen.

**Pull request closed without merging**